middlewares/validators/user.js

```diff
@@ -200,6 +200,7 @@ async function getUsers(req, res, next) {
       filterBy: joi.string().optional(),
       days: joi.string().optional(),
       dev: joi.string().optional(),
+      departed: joi.string().optional(),
```

Comment:
for `departed`, enforce it to be a boolean instead of a string (`"true"` or `"false"`).
use JOI transformations (`joi.boolean()`)

Reply:
We don’t currently have any params accepted as booleans in query strings. Adding this would need transformations in the backend to convert "true"/"false" strings to booleans. This can also be seen with the dev flag, which is a boolean but is handled as a string. It’s better to take this up as an enhancement task. Let me know your thoughts.

Reply:
Have raised this issue here - https://github.com/Real-Dev-Squad/website-backend/issues/2272